### PR TITLE
add solution

### PR DESCRIPTION
**Pull request by AndriiYelieva**
[DEMO LINK](https://AndriiYelieva.github.io/react_goods-selector-js/)

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -14,70 +16,70 @@ export const goods = [
   'Garlic',
 ];
 
-export const App = () => (
-  <main className="section container">
-    <h1 className="title is-flex is-align-items-center">No goods selected</h1>
+export const App = () => {
+  const [selectGoods, setSelectGoods] = useState('Jam');
```
Comment (etojeDenys) [inline src/App.jsx:20]:
```suggestion
  const [selectedGood, setSelectedGood] = useState('Jam');
```

File: src/App.jsx [hunk 2]
```diff
@@ -14,70 +16,70 @@ export const goods = [
...
-            Dumplings
-          </td>
-        </tr>
+            return (
+              <tr
+                data-cy="Good"
+                className={classnames({
+                  'has-background-success-light': isSelected,
+                })}
+              >
```
Comment (etojeDenys) [inline src/App.jsx:53]: don't forget about key prop

**Review by etojeDenys — CHANGES_REQUESTED** [review]
I should have the ability to remove selected Good by clicking twice on the item

https://github.com/mate-academy/react_goods-selector-js/assets/70220667/2aca9643-c625-4852-bf25-8ea8d8938a3e

File: src/App.jsx (hunk 2 above)
Comment (IvanFesenko) [reply src/App.jsx:53]: Same here add key 
2023-07-04_13h00_25

**Review by IvanFesenko — CHANGES_REQUESTED** [review]
Not fixed
2023-07-04_12h59_41

https://github.com/mate-academy/react_goods-selector-js/assets/61904995/0397716c-c2ff-46fa-a35d-3e213679be12

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**